File: C/TABUADA.c
#include <stdio.h>


int main()
{
    int N, i, multi;

    printf("Deseja a tabuada para qual valor? ");
    scanf("%d", &N);

    for (i = 1; i <= 10; i++) {
        multi = N * i;
        printf("%d X %d = %d\n", N, i, multi);

    }

    return 0;
}

File: JAVA/TABUADA.java
package exercicios4;

import java.util.Scanner;

public class TABUADA {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		System.out.print("Deseja a tabuada para qual valor? ");
		int N = sc.nextInt();
		
		
		for(int i = 1; i <= 10; i++) {
			int multi = N * i;
			System.out.println(N + " X " + i + " = " + multi);
		}
	
		sc.close();
	}

}
